pkgpanda/actions.py

```diff
@@ -88,10 +81,12 @@ def fetcher(id_, target):
     try:
         repository.add(fetcher, package_id)
     except FetchError as ex:
-        print("\nUnable to fetch package {0}: {1}".format(package_id, ex))
-        sys.exit(1)
-    sys.stdout.write("\rFetched: {0}\n".format(package_id))
-    sys.stdout.flush()
+        raise Exception("Unable to fetch package {0}: {1}".format(package_id, ex))
```

Comment:
Looks like `FetchError` could have all sorts of causes: https://github.com/dcos/dcos/blob/383c868ce55ca7b1b51f0e23525fa8bb0f767207/pkgpanda/util.py#L73 I'm not confident that we can assume it's a validation-related exception.